src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -1111,7 +1105,11 @@ def getInstalledComponents(self):
                         body = rFile.read()
 
                     for cType in self.componentTypes:
-                        if body.find("dirac-%s" % (cType)) != -1:
+                        if (
+                            body.find("dirac-%s" % (cType)) != -1
+                            or cType.lower == "service"
+                            and body.find("tornado-start-all") != -1
+                        ):
```

Comment:
When using and/or I think it's worth using parantheses to make the order of operations clear. Also we should use `in` instead of `find(xxx) == -1`:
```suggestion
                        if (
                            "dirac-%s" % (cType) in body
                            or (cType.lower == "service" and "tornado-start-all" in body)
                        ):
```